### ISSUE TITLE
Add support for windows 7 (without SP1)

### ISSUE THREAD

**Issue by MorganGeek** [keybase/client#7384]
Hi,
I have windows 7 without SP1, and I would like to run keybase client on it. 
The [installation guide of keybase](https://keybase.io/docs/the_app/install_windows) didn't mention SP1 in the requirements and I already installed the client on my Android device, but the client refuses the installation on windows since SP1 is not installed. 

I would like to know if there is a workaround for me to use keybase on  my windows 7 without proceeding to installation of SP1 ? 
If not, what are the dependencies to SP1 ? If it's needed because of [dokan](https://github.com/dokan-dev/dokany), I understand but apparently dokan is not mandatory for keybase according to https://github.com/keybase/client/issues/6690#issuecomment-297179313.

Could you provide a workaround or update your documentation with the exact requirements ? 

Thanks

**Comment by zanderz** [comment]
You are right, our installation guide is pretty light on requirements, and SP1 is only a Dokan requirement AFAIK. You could try executing the installer at the command line with `nodriver=1`, which should leave everything working except the mounted filesystem. But running such an old, updated Windows is not advisable.

**Comment by MorganGeek** [comment]
Hi @zanderz , thanks for the answer.
I've already tried running the keybase installer with `nodriver=1`, only to be advised that I should run on SP1. 
Since I don't plan to reconfigure or update my windows devices, I guess it's a dead end for me... too bad :cry: I'll probably chase alternatives soon

Thanks anyway for the advice :wink:

**Comment by zanderz** [comment]
I'll submit it to our backlog (DESKTOP-4039) and we'll make a decision whether to skip SP1 when `nodriver=1`.

**Comment by MorganGeek** [comment]
thanks ! @zanderz
